openmcyclus/DepleteReactor.py

```diff
@@ -121,13 +121,15 @@ class DepleteReactor(Facility):
     model_path = ts.String(
         doc="Path to files with the OpenMC model information",
         tooltip="Path to files with OpenMC model",
-        default="/home/abachmann/openmcyclus/tests/"
+        default="/home/abachmann/openmcyclus/examples/"
```

Comment:
It's not good practice to have a hard-coded path, such as this one. Can you generalize this path with a relative path? You can use something like `pathlib` to do this for any operating system. I don't have the exact syntax off the top of my head but it's something like

```py
from pathlib import Path

default = Path("../examples/").to_string()
```
Pathlib automatically conforms to the current OS conventions. For example, forward vs. backward slashes on UNIX or windows, respectively. The to_string method just converts it to a string, which appears to be the required type for ts.String.

Reply:
I agree. I will look into this more to avoid hard coded paths.

Reply:
So this was an interesting thing to look into. If I didn't provide a default value to the archetype class, then the path variable wasn't passed to the Depletion class correctly. But by providing the default, it was passed correctly. Seeing as how there isn't really a default for a path to where OpenMC model files should be, I decided that removing the default path was best. Therefore, I had to change each of the methods in the Depletion class to read in the path variable directly instead of instantiating the path variable with the class. Why all of this was necessary, I don't know. But the new changes in commit bec805c allows the archetype to function without specifying a default path.